docs/resources/power-query-differences.md

```diff
@@ -0,0 +1,52 @@
+
+This article provides a general overview of when you might favor one platform over the other. In general, Power Query is good for pulling and transforming data from large, external data sources and Office Scripts are good for quick, Excel-centric solutions and [Power Automate integrations](../develop/power-automate-integration.md).
+
+## Large data sources and data retrieval: Power Query
+
+We recommend Power Query when dealing with data sources from supported platforms.
+
+Power Query has [built-in data connections](https://powerquery.microsoft.com/connectors/) to hundreds of sources. Power Query is specially designed for data retrieval, transformation, and combination tasks. When you need data from one of those sources, Power Query gives you a no-code way of bringing that data into Excel in the shape you need.
+
+These Power Query connections are designed for large datasets. They do not have the same [transfer limits](../testing/platform-limits.md) as Power Automate or Excel for the web.
+
+Office Scripts offer a lightweight solution for smaller data sources or data sources not covered by Power Query connectors. This includes [using `fetch` or REST APIs](../develop/external-calls.md) or getting information from ad-hoc data sources, such as a [Teams adaptive card](../resources/scenarios/task-reminders.md).
+
+## Formatting, visualizations, and programmatic control: Office Scripts
+
+We recommend Office Scripts when your needs go beyond data importing and transformation.
+
+Nearly everything you can do manually through the Excel UI is doable with Office Scripts. They are great for applying consistent formatting to workbooks. They create charts, PivotTables, shapes, images, and other worksheet visualizations. When you're  
+
+The inclusion of TypeScript code gives you a high degree of customization. Control logic like `if...else` statements makes your script robust. This lets you do things like conditionally read data without relying on complex Excel formulas or scan the workbook for unexpected changes before changing the workbook.
+
+Formatting can be applied with Power Query through [templates](https://templates.office.com/power-query-tutorial-tm11414620). However, templates are updated at the individual or organization level, whereas Office Scripts offer more granular access control.
+
+## Power Automate integrations
+
+Office Scripts offer more options for Power Automate integration. Scripts are tailored to your solutions. You define the [input and output of the script](../develop/power-automate-integration.md#data-transfer-in-flows-for-scripts), so it works with any other connector or data in the flow.
+
+:::image type="content" source="../images/scenario-task-reminders-last-flow-step.png" alt-text="The Excel Online (Business) connector in the flow designer showing the Office Scripts option.":::
```

Comment:
For accessibility, I recommend adding a sentence describing what this screenshot illustrates. It's not immediately apparent to me. Something like, "The following screenshot shows an example of integrating..."